### ISSUE TITLE
openshift-node fails to start at installation because Docker fails

### ISSUE THREAD

**Issue by nkhare** [openshift/openshift-ansible#537]
While doing the installation from master branch (commit bac81adf9d7036e02db457eb78d5d91c3d630cd9), openshift-node fails to start at installation because Docker fails. Docker fails because following entry in */etc/sysconfig/docker* file 

```
ADD_REGISTRY='--add-registry registry.access.redhat.com --add-registry '
```
*--add-registry* needs some argument, which is not getting correctly with ansible jobs.

**Comment by sdodson** [comment]
Hmm, this should've been fixed in #516 you're sure your at the commit you referenced? Do you by chance have multiple ADD_REGISTRY lines in */etc/sysconfig/docker* ?

**Comment by nkhare** [comment]
Looks like due to some automation issue, *openshift-ansible* did not get cloned correctly. It works fine now.
